vg/vgl/canvas.go

```diff
@@ -0,0 +1,348 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package vgl
+
+import (
+	"fmt"
+	"image/color"
+	"runtime"
+	"time"
+
+	"github.com/go-gl/gl/v2.1/gl"
+	"github.com/go-gl/glfw/v3.1/glfw"
+	"github.com/gonum/plot/vg"
+	"github.com/llgcode/draw2d"
+	"github.com/llgcode/draw2d/draw2dgl"
```

Comment:
Does draw2d/draw2dgl actually use OpenGL to do rasterization? Looking at the code, it looks like it uses the freetype-go software rasterizer: https://github.com/llgcode/draw2d/blob/master/draw2dgl/gc.go#L9. Way back, I did an experiment using software rendering for interactive plotting with Plotinum. It was too slow. See: https://code.google.com/p/eaburns/source/browse/showplot/main.go.

If we provide an OpenGL implementation of vg, I think that it should use true hardware rasterization.